src/integrations/prefect-kubernetes/prefect_kubernetes/worker.py

```diff
@@ -621,7 +646,134 @@ async def run(
 
             return KubernetesWorkerResult(identifier=pid, status_code=status_code)
 
-    async def teardown(self, *exc_info):
+    async def submit(
+        self,
+        flow: "Flow[..., R]",
+        parameters: dict[str, Any] | None = None,
+        job_variables: dict[str, Any] | None = None,
+    ) -> "PrefectFlowRunFuture[R]":
+        """
+        EXPERIMENTAL: The interface for this method is subject to change.
```

Comment:
maybe this should be emitted or logged as a warning too just to be safe

Reply:
Added a warning in [d18f090](https://github.com/PrefectHQ/prefect/pull/17218/commits/d18f090343cde29cf037a7b11146ff684d7f1051)